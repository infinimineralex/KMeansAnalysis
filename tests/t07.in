2
data7.txt
7

