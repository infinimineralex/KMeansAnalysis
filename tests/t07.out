How many clusters should there be? (Write below)
What's the name of the file we're using?
Running the algorithm with this many data points: 1
What test number is this? 
The centroid coordinates are: 
Centroid #1: 
X: 10
Y: 10
Centroid #2: 
X: -nan
Y: -nan
